### PR TITLE
Ethics links

### PR DESCRIPTION
**Pull request by carriewright11**
Starting to add links to the ethics question... I wonder if we might want a new styled box for this? what do you think @avahoffman  and @ehumph ?

I will make some of these links more specific to the actual header, once we merge in the ethics branch.

### CODE REVIEW

**Comment by avahoffman** [comment]
I like the idea of a new div for this. Maybe with this icon? https://www.iconpacks.net/free-icon/justice-1957.html

**Comment by carriewright11** [comment]
> I like the idea of a new div for this. Maybe with this icon? https://www.iconpacks.net/free-icon/justice-1957.html

Love it!

File: 01-intro.Rmd [hunk 1]
```diff
@@ -72,6 +72,9 @@ The following outline several key questions:
 
 To determine if your research requires compliance with other policies that may influence how you share your data, take this [quiz](https://sharing.nih.gov/other-sharing-policies/which-policies-apply-to-my-research).
 
+<div class = "ethics"> 
+In addition to these questions, there are ethical considerations that you may want to think about. See the [ethics section](https://hutchdatascience.org/NIH_Data_Sharing/ethics.html) of the appendix for more information.
```
Comment (avahoffman) [inline 01-intro.Rmd:76]: Any reason not create a link e.g. {#ethics} ?

**Comment by ehumph** [comment]
The new box looks great! I like that the statement about ethics is set apart as well, since it's not really in the new NIH guidelines.

File: 01-intro.Rmd (hunk 1 above)
Comment (carriewright11) [reply 01-intro.Rmd:76]: just keeping it more separate kinda - but we can definitely do a link

**Comment by carriewright11** [comment]
OK I updated the links. Do I have one too many in the writing section? @avahoffman  and @ehumph

**Comment by carriewright11** [comment]
maybe we need one about data quality somewhere?

File: 01-intro.Rmd [hunk 2]
```diff
@@ -220,6 +223,10 @@ Justifiable ethical, legal, and technical factors for limiting sharing of data i
 - Restrictions are imposed by existing or anticipated agreements with other parties 
 - Datasets cannot practically be digitized with reasonable efforts
 
+<div class = "ethics"> 
+For additional information about potential ethical considerations, see the [ethics section](#ethics) of the appendix.
```
Comment (avahoffman) [inline 01-intro.Rmd:227]:
```suggestion
For additional information about potential ethical considerations, including if data is of sufficient quality to be shared, see the [ethics section](#ethics) of the appendix.
```

Comment (avahoffman) [reply 01-intro.Rmd:227]: > maybe we need one about data quality somewhere?

What about this?

**Review by avahoffman — APPROVED** [review]
Overall, looks great!

File: 01-intro.Rmd (hunk 2 above)
Comment (carriewright11) [reply 01-intro.Rmd:227]: Sure that works!

**Pull request merged**